static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -34,10 +34,14 @@ const minArticlesBeforeShowingBanner = 3;
 
 const lastClosedAtKey = 'engagementBannerLastClosedAt';
 
+/**
+ * We're temporarily using the "/rest-of-world" route
+ * for users in the "european-union" region.
+ */
 const getTimestampOfLastBannerDeployForLocation = (
     region: ReaderRevenueRegion
 ): Promise<string> =>
-    fetchJson(`/reader-revenue/contributions-banner-deploy-log/${region}`, {
+    fetchJson(`/reader-revenue/contributions-banner-deploy-log/${region === 'european-union' ? 'rest-of-world' : region}`, {
```

Comment:
Would it be worth extracting this 'url region' logic out into a separate function which could be used by both banners?

Reply:
The logic is slightly different for the two banners right now. For EU users the `subscription-banner.js` falls back to using the `united-kingdom` route (as requested by stakeholder) and the `membership-engagement-banner.js` falls back to using the `rest-of-world` route (basically as before). There's still a little repitition, but this logic is temporary and we'll hopefully restore to `/${region}` once we support a `/european-union` route.

Reply:
👍